nixos/modules/virtualisation/lxd-agent.nix

```
@@ -60,7 +60,13 @@ in {
       wantedBy = [ "multi-user.target" ];
       before = [ "shutdown.target" ];
       conflicts = [ "shutdown.target" ];
-      path = [ pkgs.kmod pkgs.util-linux ];
+      path = [
+        pkgs.kmod
+        pkgs.util-linux
+
+        # allow exec to find system binaries
+        "/run/current-system/sw"
```

```suggestion
        "/run/current-system/sw/bin"
```

bin? sbin? either? both?

---

Ah `incus exec`, so likely both.

---

These subdirectories are already appended. I know because I originally add /bin and then saw it ended up as /bin/bin :)

https://github.com/NixOS/nixpkgs/blob/5f64a12a728902226210bf01d25ec6cbb9d9265b/nixos/lib/systemd-lib.nix#L325

---

Oh, wow. Thanks for pointing that out.

---

Also, the test I added would catch at least the `bin` dir missing.